docs/SRS-Volere/SRS.tex

```diff
@@ -348,9 +348,17 @@ \subsection{Cultural Requirements}
 
 \section{Compliance Requirements}
 \subsection{Legal Requirements}
-\lips
+\begin{itemize}
+    \item The application is to strictly abide by all regulations governing
+    the treatment of water and wastewater. This includes, but is not limited 
+    to the \textit{Safe Drinking Water Act} (SDWA), 2002, the
+    \textit{Ontario Water Resources Act} (OWRA), the 
+    \textit{Environmental Protection Act} and the \textit{Clean Water Act}.
```

Comment:
These don't seem relevant to our app, these are regulations for the water treatment, but nothing about our app

Reply:
I was thinking it more so as we can't design our app with any features which would be in violation of these compliance regulations. The existing entry/exit protocols are built around adhering to these regs, and things like the requirement to generate reports come from a regulatory need to adhere to legislation.

Reply:
do the regulations listed here actually have anything to do with the entry/exit protocols?

Reply:
The entry/exit protocols would exist to adhere to the requirement in those regs to ensure the integrity of the water supply. These are the requirements that stops us from modifying the existing system in place, such as the physical logbook. 

The compliance requirements can be broad, they're just some requirement boundary that can't be crossed. Example from the template which is similar:
![image](https://github.com/user-attachments/assets/2a8b32d3-c0d8-441e-be5c-619903641794)

Reply:
Right, but that's not my question my question is if the `Safe Drinking Water Act`, `Ontario Water Resources Act`, `Environmental Protection Act`, or `Clean Water Act` are related to the entry/exit protocols of the city? and if not do they have any place here, since they are not related to our application